# HG changeset patch
# User Rik <rik@octave.org>
# Date 1447951591 28800
#      Thu Nov 19 08:46:31 2015 -0800
# Node ID b76162d57ead0374b60bd39c9f7e85a7486581cd
# Parent  7c21b151b8b43c6df0a3d99e495184b59a77c99d
munge-texi.pl: Silence warning from new versions of Perl (bug #46475).

* munge-texi.pl: Escape any brace characters in single-quoted regexp to avoid a
warning in newewer versions of Perl.

diff --git a/doc/interpreter/munge-texi.pl b/doc/interpreter/munge-texi.pl
--- a/doc/interpreter/munge-texi.pl
+++ b/doc/interpreter/munge-texi.pl
@@ -92,22 +92,22 @@ TXI_LINE: while (<STDIN>)
 ################################################################################
 sub extract_docstring
 {
   my ($docstring, $arg_list, $func_list, $repl, $rest_of_line);
 
   while (defined ($_ = <DOCFH>) and ! /$doc_delim/o)
   {
     # expand any @seealso references
-    if (m'^@seealso{')
+    if (m'^@seealso\{')
     {
       # Join multiple lines until full macro body found
       while (! /}/m) { $_ .= <DOCFH>; }
 
-      ($arg_list, $rest_of_line) = m'^@seealso{(.*)}(.*)?'s;
+      ($arg_list, $rest_of_line) = m'^@seealso\{(.*)\}(.*)?'s;
 
       $func_list = $arg_list;
       $func_list =~ s/\s+//gs;
       $repl = "";
       foreach $func (split (/,/, $func_list))
       {
         $func =~ s/^@/@@/;   # Texinfo uses @@ to produce '@'
         $repl .= "\@ref{XREF$func,,$func}, ";
